packages/suite/src/components/wallet/Discovery/index.tsx

```diff
@@ -20,17 +20,20 @@ const AccountDiscovery = (props: Props) => {
 
     return (
         <View>
-            <Button onClick={props.start} isLoading>
+            <Button
+                onClick={props.start}
+                isLoading={d.status > 0 && d.status < DISCOVERY_STATUS.IDLE}
```

Comment:
this condition always evaluates to false because DISCOVERY_STATUS.IDLE is also 0